src/index.html

```diff
@@ -3,11 +3,67 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="imac"
+      class="card__imag"
+      >
+      <h2
+      class="card__product-name">
+      APPLE A1419 iMac 27" Retina
+        <br>
+      5K Monoblock (MNED2UA/A)
+      </h2>
```

Comment:
```suggestion
      <h2
        class="card__product-name"
      >
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```
fix indentation
no need to make `br` here